maquetado/index.html

```diff
@@ -9,6 +9,16 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <h1 class="title">Front-End Tools</h1>
+    <div class="card">
+      <div class="tech">
+        <img class="logo" src="assets/react_icon.svg">
```

Comment:
I recommend you to set the attribute `alt` and change the class name for something more descriptive, for example: `tech-logo`